tegel/src/components/block/sdds-block.tsx

```diff
@@ -0,0 +1,21 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sdds-block',
+  styleUrl: 'sdds-block.scss',
+  shadow: false,
+})
+export class SddsBlock {
+  /** Mode variant of the component, based on current mode. */
+  @Prop() modeVariant: 'primary' | 'secondary' = null;
+
+  render() {
+    return (
+      <Host>
+        <div class={`sdds-block-webcomponent sdds-block-${this.modeVariant}`}>
+          <slot></slot>
+        </div>
+      </Host>
```

Comment:
What do you think of putting classes to the Host level?
I am just thinking of shorter code, nothing else.

```suggestion
      <Host class={`sdds-block-webcomponent sdds-block-${this.modeVariant}`}> 
          <slot></slot>      
      </Host>
```

Reply:
Or the second option is to drop <Host> entirely?

Reply:
Good idea! We can drop the host element all together